view/light.twig

```diff
@@ -134,7 +135,7 @@
       {% include 'left.inc' ignore missing %}
     {% endif %}
     {% block error %}
-    <div id="maincontent">
```

Comment:
Dropping the id `maincontent` will affect the contents of this element. https://github.com/NatLibFi/Skosmos/blob/5fb7bf792be2ab09b6f9928f904f1517bcfb5efe/resource/css/styles.css#L606-L608 If these lines are no longer needed or they did not do anything, I suggest removing them altogether, i.e., remove all references to `#maincontent`.

Reply:
True. I didn't see it affecting a thing but in theory, it would be there to make sure a link or subheader does not change the line heights. I'll actually just keep the id on the div in case someone using Skosmos with custom CSS would refer to it.

That being said, now we'd have main element #main, div element #maincontent, and div element #main-container.